data/mods/gen8bdsp/formats-data.ts

```diff
@@ -2033,12 +2033,12 @@ export const FormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 	giratina: {
 		tier: "Uber",
 		doublesTier: "DUber",
-		randomBattleMoves: ["hex", "willowisp", "rest", "sleeptalk"],
+		randomBattleMoves: ["shadowball", "dragonpulse", "rest", "sleeptalk", "aurasphere"],
```

Comment:
Calm mind was definitely intented here, this is just an oversight from Rusty. We didn't want giratina as hex+status since it doesn't get toxic anymore